### PR TITLE
Updated terraform issues highlighted by snyk

### PR DESCRIPTION
**Pull request by Girish-pingala**

### CODE REVIEW

**Review by ggrig — APPROVED** [review]

File: infra/terraform/aws/modules/alb/main.tf [hunk 1]
```diff
@@ -1,9 +1,10 @@
 resource "aws_lb" "default" {
   name               = "${var.project_name}-lb"
-  internal           = false
+  internal           = true
```
Comment (pranaysahith) [inline infra/terraform/aws/modules/alb/main.tf:3]: This should not be internal, as the loadbalancer should be accessible from outside.

**Review by pranaysahith — CHANGES_REQUESTED** [review]

**Comment by chrisdiao** [comment]
@Girish-pingala could you please:
1. Add "HOW TO TEST" information in this PR:
     UNIT TEST
    INTEGRATION TEST
    SMOKE TEST

2. Update the DOCUMENTATION - if applicable

File: infra/terraform/aws/modules/alb/main.tf (hunk 1 above)
Comment (Girish-pingala) [reply infra/terraform/aws/modules/alb/main.tf]: @pranaysahith Changes has been rolled back. Internal is set to false.

**Comment by chrisdiao** [comment]
@Girish-pingala , could you resolve the conflict for this PR, please?

**Review by pranaysahith — APPROVED** [review]

**Pull request merged**